build_deploy_dbs.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+DOCKER_CONF="$PWD/.docker"
+mkdir -p "$DOCKER_CONF"
+docker --config="$DOCKER_CONF" login -u="$QUAY_USER" -p="$QUAY_TOKEN" quay.io
```

Comment:
just be aware that any ci node this runs on is going to persist this to disk, which can add concurrency woes as well as may not be ideal when it comes to secrets at rest. this is probably okay in this context as long as $PWD/.nnn is the job build path in terms of concurrency.